search-sentiments-map/public/scripts/map-script.js

```diff
@@ -154,15 +178,24 @@ function styleFeature(feature) {
  * @param {?google.maps.MouseEvent} e Mouse-in event.
  */
 function mouseInToRegion(e) {
-  // Set the hover country so the setStyle function can change the border.
-  e.feature.setProperty('country', 'hover');
-
+  let countryData = e.feature.getProperty('country_data');
   // Add popup info window with country info.
-  const countryInfo = e.feature.getProperty('name') + ': ' +
-      e.feature.getProperty('country_data').toLocaleString();
-  infowindow.setContent(countryInfo);
-  infowindow.setPosition(e.latLng);
-  infowindow.open(map);
+  if (countryData != null) {
```

Comment:
if (countryData == null) {
  return;
}
//blahblahblah

This way it can say you a huge {
} blob.